Updated Privacy policy 🔏 (#3280)
libre-tube/LibreTube: PRIVACY_POLICY.md

<h2 align="left">
<sub>
<img  src="fastlane/metadata/android/en-US/images/readme/privacy.svg"
      height="30"
      width="30">
</sub>

> The LibreTube project - Privacy Policy

Reviewer: ```suggestion
LibreTube - Privacy Policy
```

</h2>

> ``At LibreTube, we are committed to protecting the privacy of our users. This Privacy Policy outlines the types of information application collect, how we use it, and how it protect your personal information.``

Reviewer: ```suggestion
``LibreTube aims to protect the privacy of its users. [Our Privacy Policy](/PRIVACY_POLICY.md) gives detailed information on which data the app stores in order to work, how it is being used, and how the project protects your personal information. It is recommended to read the privacy policy of LibreTube as well as the privacy policy of the instance you have chosen inside the app.``
```

### Information Collection
LibreTube only collects necessary data such as watch history, settings, and bookmarks. Users can choose to turn off data collection features. The app does not collect any data about how it is used or profile its users, and app also avoids using libraries that could track usage data.

### Information Use

> LibreTube keeps your data only on your device and does not share it with anyone else. Your data is protected by encryption to prevent unauthorized access.

Reviewer: ```suggestion
LibreTube doesn't share your data with 3rd parties. If you use the app without being logged in with an account, no relevant data will be shared with the Piped servers at all, it'll only be stored locally on the device. When being logged in with a Piped account, your subscriptions and playlists are saved to your Piped account.
```

### Open Source Software
LibreTube is an open source software that is built for learning and research purposes. We welcome and encourage collaboration and contributions from the community to help improve the app.

> ### Source Instances

Reviewer: ```suggestion
### Instance list source
```

### Instances & Data handling
User data is not shared with third parties, including Piped instances. Local playlists are stored on the app and not shared with Piped. There are no logs of user activity or accessed information kept by the app.

### Piped account
Piped accounts can be completely deleted with no history kept, providing increased security for users on the site.

### Data Security
LibreTube connects to Piped instances instead of directly to YouTube, providing an additional layer of privacy. However, trust is shifted to the instance owner. Piped instances do not log user-specific data, but there is no guarantee regarding the actions of the instance owner. Even if logging occurred, the data would be less than that collected by YouTube since LibreTube does not send telemetry data to the instance's server.

### Disclaimer
The LibreTube project is not affiliated, authorized, or endorsed by YouTube, Google LLC, or any of its affiliates or subsidiaries. Any intellectual property used is owned by the respective owners.

### Changes to Privacy Policy

> By using the LibreTube app, you consent to the handling of your information as outlined in this Privacy Policy. If you have any questions or concerns about our Privacy Policy, you are free to ask us question.

Reviewer: ```suggestion
By using the LibreTube app, you consent to the handling of your information as outlined in this Privacy Policy. If you have any questions or concerns about our Privacy Policy, you are free to open an issue to ask about it or ask at the official Matrix room of the app.
```
